DFReader.py

```diff
@@ -11,9 +11,14 @@
 from builtins import range
 from builtins import object
 
-import struct, time, os
+import struct
 from . import mavutil
 
+try:
```

Comment:
Are we going to have to carry this forever-more?

There's not future thing we can use to fix this instead?